src/frontend/apps/standalone_site/package.json

```diff
@@ -12,28 +12,34 @@
     "@types/react-router-dom": "5.3.3",
     "babel-jest": "29.5.0",
     "cross-env": "7.0.3",
+    "eslint": "*",
+    "eslint-config-marsha": "*",
     "fetch-mock": "9.11.0",
-    "typescript": "4.9.5"
+    "typescript": "*"
   },
   "dependencies": {
+    "@craco/craco": "7.1.0",
     "grommet": "*",
     "http-proxy-middleware": "2.0.6",
+    "lib-video": "4.0.0-beta.18",
+    "marsha-config": "*",
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-hot-toast": "2.4.0",
     "react-query": "3.39.3",
     "react-router-dom": "5.3.0",
     "react-scripts": "5.0.1",
     "resize-observer-polyfill": "^1.5.1",
-    "web-vitals": "3.3.0"
+    "web-vitals": "3.3.0",
+    "zustand": "*"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "craco start",
+    "build": "craco build",
     "postbuild": "./postbuild.sh",
-    "test": "cross-env CI=true TZ=UTC react-scripts test",
+    "test": "cross-env CI=true TZ=UTC craco test",
     "eject": "react-scripts eject",
```

Comment:
Do we keep this command ? We know it exists and we are working on how to not eject the project.